phpstan.neon.dist

```diff
@@ -1,11 +1,12 @@
 parameters:
   autoload_files:
     - vendor/autoload.php
+    - bin/.phpunit/phpunit-8.3-0/vendor/autoload.php
   paths:
     - src
     - tests
   symfony:
-    container_xml_path: '%rootDir%/../../../var/cache/dev/srcApp_KernelDevDebugContainer.xml'
+    container_xml_path: 'var/cache/test/srcApp_KernelTestDebugContainer.xml'
```

Comment:
why getting rid of the `%rootDir%`  ?

Reply:
It's not needed to configure it like that, works with this syntax as well

Reply:
until you decide to change your cache folder location :)